cmd/dep/main.go

```diff
@@ -143,19 +146,16 @@ func (c *Config) Run() (exitCode int) {
 			}
 
 			// Set up the dep context.
-			ctx := &dep.Ctx{
-				Out:     log.New(c.Stdout, "", 0),
-				Err:     errLogger,
-				Verbose: *verbose,
-			}
-			gopaths := filepath.SplitList(getEnv(c.Env, "GOPATH"))
-			err := ctx.SetPaths(c.WorkingDir, gopaths...)
-			if err != nil {
-				errLogger.Printf("%q not in any GOPATH: %s\n", c.WorkingDir, err)
+			gopaths := getGOPATHs(c.Env)
```

Comment:
I think this func, and its related helpers (`getEnv()`, `defaultGOPATH()`), , should probably move back into `github.com/golang/dep`, and be accessed indirectly either through `dep.NewContext()`, or by bringing back the `ctx.SetPaths()` function (and perhaps going back to not having a `dep.NewContext()`).

We need these funcs in an importable package, because without them, future dependers on dep won't be able to construct a `dep.Ctx` correctly.